### PR TITLE
Upgrade Loki version to 0.1.0

### PR DESCRIPTION
**Pull request by igor-karpukhin**
Upgrade Loki version to 0.1.0

### CODE REVIEW

File: resources/logging/templates/loki/statefulset.yaml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            - name: config
+              mountPath: /etc/loki
+            - name: storage
+              mountPath: "/data"
+              subPath: {{ .Values.loki.persistence.subPath }}
+          ports:
+            - name: http
+              containerPort: {{ .Values.loki.config.server.http_listen_port }}
+              protocol: TCP
+                    {{/*
```
Comment (hisarbalik) [inline resources/logging/templates/loki/statefulset.yaml]: please add here liveness and readiness check with commands options

File: resources/logging/templates/loki/deployment.yaml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+{{/*
```
Comment (hisarbalik) [inline resources/logging/templates/loki/deployment.yaml]: this file can be removed completely, it's replaced with statefulset.yaml

File: resources/logging/templates/loki/statefulset.yaml (hunk 1 above)
Comment (igor-karpukhin) [reply resources/logging/templates/loki/statefulset.yaml]: Fixed

File: resources/logging/templates/loki/deployment.yaml (hunk 2 above)
Comment (igor-karpukhin) [reply resources/logging/templates/loki/deployment.yaml]: Fixed

**Pull request merged**